docs/dietpi_docs_todo.md

```diff
@@ -144,7 +174,7 @@ Proposal:
 - Harmonize naming
 - List in alphabetical order
 
-Decision?
+#### Decision?
```

Comment:
I agree, indeed I changed the names a bid in the main navigation but not in the `dietpi_optimised_software` headers, sorry for that 😅.

Reply:
Proposal: Decision in telecon.